test/staging/Intl402/Temporal/old/instant-toLocaleString.js

```diff
@@ -7,6 +7,12 @@ description: Instant.toLocaleString()
 features: [Temporal]
 ---*/
 
+// Tolerate implementation variance by expecting consistency without being prescriptive.
+const usDayPeriodSpace =
+  new Intl.DateTimeFormat("en-US", { timeStyle: "short" })
```

Comment:
Shouldn't this be used in the first assertion, or are we looking at a difference between "en-US" and "en"?
```diff
-assert.sameValue(`${instant.toLocaleString("en", { timeZone: "America/New_York" })}`, "11/18/1976, 9:23:30 AM");
+assert.sameValue(`${instant.toLocaleString("en", { timeZone: "America/New_York" })}`, `11/18/1976, 9:23:30${usDayPeriodSpace}AM`);
```

Reply:
Good catch. It looks like there's a mix in these tests between "en" & "de" vs. "en-US" and "de-AT", which I assume was an oversight from whoever originally wrote these tests. I'll just normalize them all into "en-US" and "de-AT" throughout all these files, and I'll put that into a separate commit to keep the noise down.